modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx

```diff
@@ -171,6 +171,18 @@ const ActivityClaimPanel = ({
 							</Link>
 						</div>
 					</div>
+
+					<PRMFormik.Field
+						component={PRMForm.DragAndDrop}
+						description="Drag and drop your files here to upload."
+						label="All Contents"
+						onAccept={(value: File) => 
+							setFieldValue(
+								`activities[${activityIndex}].contents`,
+								value
+							)
+						}
```

Comment:
```suggestion
						onAccept={(value: File[]) => 
							setFieldValue(
								`activities[${activityIndex}].contents`,
								value
							)
						}
```

Nao seria um array de files? Visto que pode adicionar mais de um.